### DISCUSSION TITLE
Maybe mistake in tutorials?

### DISCUSSION THREAD

**Discussion by mudrole1** [BehaviorTree/BehaviorTree.CPP#676] [Q&A]
Hi, I have just been recently revisiting the tutorials and I think I either spotted a mistake or my understanding is wrong!

The section I am going to refer to is in Decorators on this page: https://www.behaviortree.dev/docs/learn-the-basics/BT_basics

There is a picture about opening the door. In particularly, I wonder about the fallback subtree:
```
<Fallback>
  <Inverter>
   <IsDoorOpen/>
 </Inverter>
<RetryUntilSuccessful n=5>
  <OpenDoor/>
</RetryUntilSuccessful>
</Fallback>

```
I belive that either the Inverter should be removed, or the condition node should be "IsDoorClosed". 

My reasoning:
if condition node "IsDoorOpen" return SUCCESS, the inverter will change it to "FAILURE", which will result that the fallback triggers the next strategy which is to try to open the door... but the door is opened in the first place.

**Comment by bijoua29** [comment]
Yeah, I agree that is the case. Seems like the inverter is unnecessary

**Comment by HenrikNilssonCode** [comment]
A quick look seems to confirm this. Maybe open an issue on the btcpp_website repo instead. [This issue](https://github.com/BehaviorTree/btcpp_website/issues/16) that has already been posted there seems to be related.
